src/export/Package.ts

```diff
@@ -35,9 +35,29 @@ export class Package implements Fishable {
       switch (type) {
         case Type.Profile:
           def = this.profiles.find(p => p.id === item || p.name === item || p.url === item);
+          if (!def) {
+            def = this.instances.find(
+              i =>
+                (i.id === item || i._instanceMeta.name === item || i.url === item) &&
+                i._instanceMeta.usage === 'Definition' &&
+                i.resourceType === 'StructureDefinition' &&
+                i.derivation === 'constraint' &&
+                i.type !== 'Extension'
```

Comment:
I don't know how much of a difference this will make in reality, but... since fishing is fairly common, I think it's best to make the find function as efficient as possible.  To that end, do you think it makes sense to rearrange the conditional clause so that the rarest bits are checked _first_?  For example, in the case above, I might check `i.derivation === 'constraint'` first under the assumption that most instances are _not_ profiles or extensions, so we bail out quickly rather than checking 3 separate fields for the id first.

That said, ids are generally unique, so I can also see why you might want to check those first.  So I'm not 100% positive about what is best -- but I _think_ that by putting derivation first, you'll only need one equality check in most cases (vs 3 equality checks).

Reply:
... but this may be so trivial compared to all the other things that happen that maybe it doesn't matter.  I don't know.  If you think it doesn't make sense to change this (and the similar cases below) then I won't argue with you.